### DISCUSSION TITLE
Обновление файловой системы по воздуху

### DISCUSSION THREAD

**Discussion by Sculptonics** [vvip-68/LedPanelWiFi#6] [Вопросы и ответы] [answered]
Здравствуйте!
После загрузки LittleFS через сетевой порт контроллер создаёт точку доступа PanelAP. Подключение к этой точке доступа проходит без проблем. Но страница по адресу 192.168.4.1 не доступна.
Что необходимо сделать для успешного обновления? Очень хочется "на лету" добавлять эффекты, без демонтажа платы из замурованной коробки.
Ядро и библиотеки - рекомендованные. Работа после прошивки через COM порт без сбоев.

**Comment by vvip-68** [comment]
Ну как минимум нужно смотреть что пишет в мониторе порта контроллера. Если прошивка залилась и работает, файловая система загружена, подключение к контроллеру есть - я не вижу причин, чтобы страница была недоступна.

**Comment by vvip-68** [comment, marked as answer]
>В момент вызова страницы в COM порте следующее:
web -> '/web/index.html.gz' -> send (text/html) web -> '/web/styles.7e4222ebe23cabfb.css.gz' -> send (text/css) FM: 14984 - 11984 web -> '/web/runtime.7882e08c7503fb34.js.gz' -> send (text/javascript) FM: 14184 - 12784 web -> '/web/pf.dcddd532a07c6b2d.js.gz' -> send (text/javascript) FM: 10768 - 7768 web -> '/web/main.e44c7c6a45b2d807.js.gz' -> send (text/javascript) FM: 5736 - 5936
Спинер страницы крутится, пока, наконец, браузер не приостанавливает загрузку. Пробовал разные устройства и браузеры.

Ну, судя по логу - устройство отправило все компоненты страницы браузеру.
Тогда нужно смотреть в браузерном devTools (в браузере на компе - F12) - на закладке Network статус запросов - получено / нет и состав страницы на закладке Elements
